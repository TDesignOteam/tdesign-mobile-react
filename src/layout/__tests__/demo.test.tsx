import { describe, expect, render, it } from '@test/utils';

import baseDemo from '../_example/base';
import indexDemo from '../_example/index';
import offsetDemo from '../_example/offset';

const mapper = {
  baseDemo,
  indexDemo,
  offsetDemo,
};

describe('Col', () => {
  Object.keys(mapper).forEach((demoName) => {
    it(`Col ${demoName} demo works fine`, () => {
      const { container } = render(mapper[demoName]);

      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
