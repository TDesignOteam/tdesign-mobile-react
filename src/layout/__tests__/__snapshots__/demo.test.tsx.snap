// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Col > Col baseDemo demo works fine 1`] = `null`;

exports[`Col > Col indexDemo demo works fine 1`] = `null`;

exports[`Col > Col offsetDemo demo works fine 1`] = `null`;
